<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketWidget</class>
 <widget class="QDialog" name="TicketWidget">
  <property name="windowTitle"><string>Ticket</string></property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item><widget class="QTextEdit" name="textTicket"><property name="readOnly"><bool>true</bool></property></widget></item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox">
     <item><widget class="QPushButton" name="btnClose"><property name="text"><string>Fermer</string></property></widget></item>
     <item><widget class="QPushButton" name="btnPrint"><property name="text"><string>Imprimer en PDF</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>